### ISSUE TITLE
"Contact" does not redirect the user to a place

### ISSUE THREAD

**Issue by mohsenkhosroanjam** [NoshonNetworks/landver#265]
**Description:**
As a user, I can click on the "Contact" option to see the contact information of the website owners.

**Actual result:**
When I click on this option, I am not redirected to any page or section that contains contact information. 

**Expected result:**
When user clicks on the "Contact" option, they should be given the contact information. 

Steps to reproduce:
1- Go to https://www.landver.net/
2- Scroll down until the end of the page
3- Click on "Contact" option

URL: https://www.landver.net/#contact
Screenshot:
![image](https://github.com/user-attachments/assets/ac74f6f7-c50c-4e5d-b44f-9e3e35d6b408)

**Comment by Prosper1218** [comment]
hello, I'm Prosper williams. I'm a front-end developer with years of experience. I have experience working with frontend tools like nextJs, typescript, javascript I also have experience working with tailwind CSS. I have gone through your project's folder here on Git Hub and fished out what the problem is, so I'm asking you to let me fix this for you. I'll be sure to work closely with you to know how you want the contact information displayed. I'll send a pull request in 4hours.
thanks.

**Comment by OWK50GA** [comment]
Hi, @mohsenkhosroanjam
I've checked this on my own.
I don't know, but this seems to be handled already.
The reason for not seeing it is that the contact hashtag is put on a div that is very close to the point where these navigation options are. That's why you aren't seeing it

This is the view for the about section. 

I've not checked the code, but it's probably just a CTA to send a mail
However, if you want a special page for contacts, or more things added to the section, then I am willing to handle this issue. 

![Screenshot_2024-12-26-16-48-48-174_com android chrome](https://github.com/user-attachments/assets/aead356e-832a-4882-a11f-c027a98863ac)

**Comment by mohsenkhosroanjam** [comment]
@OWK50GA  Thank you so much for checking and explaining. As an end-user I am confused working with Contact option. I think it is a good idea to change "Newsletter" to "Contact" if it has been supposed to work as a Contact option. What do you think?

**Comment by OWK50GA** [comment]
@mohsenkhosroanjam
Yes, I think that would be ideal, changing the contact option to "Newsletter". Perhaps even a CTA like "Subscribe to Newsletter", if that's not too long, could also be good.

I'm willing to handle it if necessary 😃

**Comment by mohsenkhosroanjam** [comment]
@OWK50GA Seems good. Do I have permission to assign this issue to you? Or I should ask the project owners to assign?

**Comment by OWK50GA** [comment]
@mohsenkhosroanjam, Sure
Ask the project owners to assign, thank you 😊
